Gemfile

```diff
@@ -6,8 +6,19 @@ group :test do
   gem 'rspec'
   gem 'simplecov', require: false
   gem 'simplecov-console', require: false
+  gem 'capybara'
+  gem 'sinatra'
+  gem 'sinatra-reloader'
+  gem 'sinatra-contrib'
+  gem 'rack'
+  gem "thin"
+  gem "puma"
+  gem 'reel'
+  gem 'http'
+  gem 'webrick'
```

Comment:
Many GEMs, would just question and maybe think if they are all necessary

Reply:
I agree I probably don't require all of these but I had many gem issues in the Bookmarks challenge this week so added more to cover all bases. In future, I need to make more of a concerted effort to only add relevant gems in future.